### PR TITLE
Some simplifications

### PR DESCRIPTION
**Pull request by xristos1994**
This PR is related to this task: [Several Fixes and simplifications](https://trello.com/c/wPMQv3wL/35-several-fixes-and-simplifications)

### CODE REVIEW

File: src/cms/cms-utils.js [hunk 1]
```diff
@@ -10,5 +12,115 @@ if (typeof window !== 'undefined') {
 
   document.head.appendChild(style);
 
-  console.log('Style added');
+  const addHelperInfo = () => {
+    // NOTE: If we are in a training editor, we want to display some info about the training
+    if (window.location.href.includes('collections/trainings/entries/')) {
+      // FIXME This interval is here because the content of the page is not available from the
+      // begining. Check if we could find a way to execute the function when content is ready.
```
Comment (akounoupi) [inline src/cms/cms-utils.js:19]: We could maybe use an IntersectionObserver to know when the element is in the view

File: src/cms/cms-utils.js [hunk 2]
```diff
@@ -10,5 +12,115 @@ if (typeof window !== 'undefined') {
...
+          infoElement.style = 'margin-bottom: 16px; padding: 8px; background-color: #ffff5f; color: #000; border: 2px solid rgb(223, 223, 227); border-radius: 5px';
+          infoElement.className = 'info-element';
+
+          infoElement.append('You will find this training at: ');
+          infoElement.appendChild(trainingUrl);
+
+          infoElement.appendChild(document.createElement('br'));
+
+          infoElement.append('You can edit this training at: ');
+          infoElement.appendChild(adminUrl);
```
Comment (akounoupi) [inline src/cms/cms-utils.js:57]: just a suggestion, we could use template strings to avoid multiple appends

File: src/cms/cms-utils.js (hunk 1 above)
Comment (xristos1994) [reply src/cms/cms-utils.js:19]: The issue here is not that the element is somewhere in the DOM but not visible. It's that the element is not inside the DOM and it is added asynchronously. So, what would we observe?

File: src/cms/cms-utils.js (hunk 2 above)
Comment (xristos1994) [reply src/cms/cms-utils.js:57]: It is not bad to have multiple appends. And using this API is a bit more safe than writing html inside strings. 
There may be better solutions here but the whole implementation is a big and dangerous hack. So, I suggest to let this discussion for when we revisit this file.

File: src/cms/cms-utils.js (hunk 1 above)
Comment (akounoupi) [reply src/cms/cms-utils.js:19]: In this case we could use a MutationObserver like in this example  https://usefulangle.com/post/356/javascript-detect-element-added-to-dom

Comment (xristos1994) [reply src/cms/cms-utils.js]: Done

**Review by anastasiakounoupi — DISMISSED** [review]

**Pull request merged**